internal/provisioner/plain/main.go

```diff
@@ -175,14 +175,7 @@ func main() {
 			PodNamespace:    ns,
 			UnpackImage:     unpackImage,
 		},
-		rukpakv1alpha1.SourceTypeGit: &source.Git{
-			Client:          mgr.GetClient(),
-			KubeClient:      kubeClient,
-			ProvisionerName: plainBundleProvisionerName,
-			PodNamespace:    ns,
-			UnpackImage:     unpackImage,
-			GitClientImage:  gitClientImage,
-		},
+		rukpakv1alpha1.SourceTypeGit: &source.Git{},
```

Comment:
nit: one other thing I realized is we can remove the git client image `StringVar` from the command line options in line 82 since we no longer rely on an external git image to resolve git references.

Reply:
I missed it.  I'll take it out.  Thanks.